### ISSUE TITLE
Implement system `acadsin-zho-hani-latn-2002` (Academica Sinica -- Chinese Tongyong Pinyin (2002)

### ISSUE THREAD

**Issue by ronaldtse** [interscript/interscript-ruby#194]
This issue is to implement the transliteration system of acadsin-zho-hani-latn-2002.

This system is referred in the GeoNames database as `zho_Hani2Latn_AcadSin_2002`, with the system title 'Academica Sinica -- Chinese Tongyong Pinyin (2002)'.

 Tests should rely on the data extracted for the `zho_Hani2Latn_AcadSin_2002` system in https://github.com/riboseinc/geonames-transliteration-data .

**Comment by ronaldtse** [comment]
For @chaaklau 

code: acadsin-zho-hani-latn-2002
authority: acadsin
language: zho
source script: hani
dest script: latn
id: 2002

**Comment by chaaklau** [comment]
There are only two entries in `geonames-transliteration-data`

```
TRANSL_CD,SRC_UNI,SRC_NT,SRC_FULL_NAME_RO,SRC_FULL_NAME_RG,DEST_UNI,DEST_NT,DEST_FULL_NAME_RO,DEST_FULL_NAME_RG,LC
zho_Hani2Latn_AcadSin_2002,-2628435,N,Fukang,Fukang,14221806,NS,"阜康","阜康",zho
zho_Hani2Latn_AcadSin_2002,14347812,N,"Bositeng Hu","Bositeng Hu",14347813,NS,"博斯腾湖","博斯腾湖",zho
```

More testing may be needed.

**Comment by chaaklau** [comment]
Added to this pull-request 
#199

**Comment by ronaldtse** [comment]
Closed by #199
